utils/display.go

```diff
@@ -922,6 +922,62 @@ func AppTable(app *meroxa.Application) string {
 	return mainTable.String()
 }
 
+func BuildsTable(builds []*meroxa.Build, hideHeaders bool) string {
+	if len(builds) == 0 {
+		return ""
+	}
+
+	table := simpletable.New()
+	if !hideHeaders {
+		table.Header = &simpletable.Header{
+			Cells: []*simpletable.Cell{
+				{Align: simpletable.AlignCenter, Text: "UUID"},
+				{Align: simpletable.AlignCenter, Text: "STATE"},
+				{Align: simpletable.AlignCenter, Text: "DETAILS"},
+				{Align: simpletable.AlignCenter, Text: "CREATED AT"},
+			},
+		}
+	}
+
+	for _, p := range builds {
+		r := []*simpletable.Cell{
+			{Align: simpletable.AlignRight, Text: p.Uuid},
+			{Align: simpletable.AlignCenter, Text: p.Status.State},
+			{Align: simpletable.AlignCenter, Text: p.Status.Details},
+			{Align: simpletable.AlignCenter, Text: p.UpdatedAt},
```

Comment:
the heading is created, and the value is updated.  but i think we would want both anyway